src/Administration/Resources/app/administration/src/app/component/data-grid/sw-data-grid/index.js

```diff
@@ -198,10 +198,18 @@ Component.register('sw-data-grid', {
             };
         },
 
-        localStorageItemKey() {
+        localStorageGridColumnsItemKey() {
```

Comment:
We cannot rename methods. Please let the old method there and deprecate it

Reply:
Did not know that, thanks for the heads up. Fixed!